amplicon-pooling.ipynb

```diff
@@ -0,0 +1,1063 @@
+       "      <td>109379Z</td>\n",
+       "      <td>2021-08-17</td>\n",
+       "      <td>978215</td>\n",
+       "      <td>RNBJ0628</td>\n",
+       "      <td>Echo550</td>\n",
+       "      <td>THDMI_UK_Plate_3</td>\n",
+       "      <td>THDMI UK</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>3</td>\n",
+       "      <td>3</td>\n",
+       "      <td>MB_SF</td>\n",
+       "      <td>166032128</td>\n",
+       "      <td>Carmen_HOWE_KF3</td>\n",
+       "      <td>109379Z</td>\n",
+       "      <td>2021-08-17</td>\n",
+       "      <td>978215</td>\n",
+       "      <td>RNBJ0628</td>\n",
+       "      <td>Echo550</td>\n",
+       "      <td>THDMI_UK_Plate_4</td>\n",
+       "      <td>THDMI UK</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "      <td>AS</td>\n",
+       "      <td>166032128</td>\n",
+       "      <td>Carmen_HOWE_KF4</td>\n",
+       "      <td>109379Z</td>\n",
+       "      <td>2021-08-17</td>\n",
+       "      <td>978215</td>\n",
+       "      <td>RNBJ0628</td>\n",
+       "      <td>Echo550</td>\n",
+       "      <td>THDMI_US_Plate_6</td>\n",
+       "      <td>THDMI US</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  Plate Position Primer Plate # Plating Extraction Kit Lot Extraction Robot  \\\n",
+       "0              1              1      SF          166032128  Carmen_HOWE_KF3   \n",
+       "1              2              2      AS          166032128  Carmen_HOWE_KF4   \n",
+       "2              3              3   MB_SF          166032128  Carmen_HOWE_KF3   \n",
+       "3              4              4      AS          166032128  Carmen_HOWE_KF4   \n",
+       "\n",
+       "  TM1000 8 Tool Primer Date MasterMix Lot Water Lot Processing Robot  \\\n",
+       "0       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "1       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "2       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "3       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "\n",
+       "       Sample Plate Project_Name Original Name  \n",
+       "0  THDMI_UK_Plate_2     THDMI UK                \n",
+       "1  THDMI_UK_Plate_3     THDMI UK                \n",
+       "2  THDMI_UK_Plate_4     THDMI UK                \n",
+       "3  THDMI_US_Plate_6     THDMI US                "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "_metadata = [\n",
+    "    {\n",
+    "        # top left plate\n",
+    "        'Plate Position': '1',\n",
```

Comment:
Do 'Primer Plate #' and 'Plate Position' always match? If yes, do we need 2 separate values?

Reply:
They don't always match, although in practice most of the times they do. `Primer Plate #` represents the plate of EMP primers that you want to use to barcode this `Plate Position`. Each 384 well plate has 4 positions (top left, top right, bottom left, and bottom right) so you get to pick 4 primer plates from available EMP primers. If a run has only 4 96-well plates it is likely that only the first four primer plates will be used. However, if there are 6 96-well plates in a run, the first group of four plates will have a match between the plate position and the primer plate. And the second group of 2 plates will likely need primer plate 5 for position 1 and primer plate 6 for position 2. Otherwise if we used again primer plate 1 and 2 it would be impossible to differentiate the samples in the first two 96-well plates vs the last two 96-well plates.